guides/common/attributes-satellite.adoc

```diff
@@ -27,6 +27,7 @@
 :APIDocURL: {BaseURL}api_guide/index#
 :HammerDocURL: {BaseURL}hammer_cli_guide/index#
 :ConfiguringVMSubscriptionsDocURL: {BaseURL}configuring_virtual_machine_subscriptions_in_red_hat_satellite/index#
+:UpdatingDocURL: {BaseURL}updating_server/index#
```

Comment:
```suggestion
:UpdatingDocURL: {BaseURL}updating_project/index#
```
You update the whole installation, i.e. the whole project. Pls also change the folder name.

Reply:
Addressed :+1: